src/BeeNet/BeeNodeClient.cs

```diff
@@ -58,6 +69,7 @@ protected virtual void Dispose(bool disposing)
 
 
         // Properties.
+        public ClientVersions ClientVersion { get; }
```

Comment:
It is never assigned. It only contains default `0` value, that is `ClientVersions.v1_4`